src/System.ComponentModel.Composition/tests/System.ComponentModel.Composition.Test.Assembly/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Would you mind renaming to System.ComponentModel.Composition.Noop.Assembly?

Reply:
you right

Reply:
@safern done